SofaKernel/framework/sofa/helper/system/FileMonitor_windows.cpp

```diff
@@ -41,6 +39,10 @@ using sofa::helper::system::thread::ctime_t ;
 #include <windows.h>
 #include <FileAPI.h>
 
+using namespace std; 
+// needs to be declared AFTER windows header file 
+//(https://developercommunity.visualstudio.com/content/problem/93889/error-c2872-byte-ambiguous-symbol.html)
```

Comment:
Couldn't this be removed instead?

Reply:
could be.
I allow using namespace std because it is in a cpp; but obviously it can be removed